### PR TITLE
Bump inkscape to 0.91

### PR DESCRIPTION
**Pull request by alarcher**
After libcdr.

### CODE REVIEW

File: components/image/inkscape/Makefile [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
```
Comment (pyhalov) [inline components/image/inkscape/Makefile]: typo in header

File: components/image/inkscape/Makefile [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+COMPONENT_CLASSIFICATION=	Applications/Graphics and Imaging
+COMPONENT_SUMMARY=	Opensource professional vector graphics editor
+COMPONENT_PROJECT_URL=	http://www.inkscape.org/
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:4d901f8a9e1924404e797ad23b8b0c495a9d155448816d95a55974314e1f141b
+COMPONENT_ARCHIVE_URL=	http://inkscape.org/en/gallery/item/3860/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE = GPLv2
+COMPONENT_LICENSE_FILE = $(COMPONENT_NAME).license
```
Comment (xen0l) [inline components/image/inkscape/Makefile]: Just a little hint - I made COMPONENT_LICENSE_FILE default to $(COMPONENT_NAME).license in https://github.com/OpenIndiana/oi-userland/commit/fd5ab52f5aebb38d382ee662954fb9e82e1da98a, so if you don't change it, you don't have to add that line. I fine either way,

**Comment by pyhalov** [comment]
please, merge them ( I mean commits )

**Comment by alarcher** [comment]
Updated.

**Comment by alarcher** [comment]
On my machine inkscape performs much better than previous version: no cpu burn at start and faster loading of svg. I would be curious to know about your experience if you get the chance to test it.

**Pull request merged**